src/stylesheets/components/_accordions.scss

```diff
@@ -1,16 +1,12 @@
 // Variables
 
-$accordion-border: 3px solid $color-gray-lightest;
+$accordion-border: 3px solid color('gray-5');
 
 // Accordion Styles
 
-$icon-size: $spacing-md-small;
-$h-padding: $spacing-medium;
-$v-padding: $spacing-md-small;
-
 @mixin accordion-list-styles {
   @include unstyled-list;
-  color: $color-base;
+  color: $color-base; // what is this in the new system
```

Comment:
In the old system, this corresponded to the darkest base color or what was off-black. In 2.0, I think this is `$theme-color-ink` which is base darkest.